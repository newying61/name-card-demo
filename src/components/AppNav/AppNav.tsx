import React from 'react';
import { Nav, StyledNavLink } from '../Styled';

const AppNav: React.FC = React.memo(() => {
  return (
    <Nav>
      <StyledNavLink to='/cards'>All cards</StyledNavLink>
      <StyledNavLink to='/add'>Add card</StyledNavLink>
    </Nav>
);
});

export default AppNav;
