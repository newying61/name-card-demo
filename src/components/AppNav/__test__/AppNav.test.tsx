import React from 'react';
import { MemoryRouter } from 'react-router';
import { render, cleanup } from '@testing-library/react';
import AppNav from '../';

afterEach(cleanup);

describe('AppNav', () => {
    it('should render without crash', () => {
      const { container } = render(
        <MemoryRouter>
          <AppNav />
        </MemoryRouter>
      );

      expect(container.firstChild).toMatchSnapshot();
  });
});