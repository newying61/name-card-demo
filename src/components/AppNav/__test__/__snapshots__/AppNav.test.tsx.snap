// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNav should render without crash 1`] = `
<nav
  class="sc-bxivhb gCIOfD"
>
  <a
    class="sc-ifAKCX jvljLJ"
    href="/cards"
  >
    All cards
  </a>
  <a
    class="sc-ifAKCX jvljLJ"
    href="/add"
  >
    Add card
  </a>
</nav>
`;
